### PR TITLE
Fix database install details for .env file

### PR DESCRIPTION
**Pull request by cmacdonell**
I think 'ushahidi' and 'username' were reversed. Also putting username and
password next to eachother makes more sense

### CODE REVIEW

File: install/installing-on-linux.md [hunk 1]
```diff
@@ -69,10 +69,10 @@ Now create a `.env` config file in the base of repository. Make sure that the da
 
 ```
 DB_HOST=localhost
-DB_NAME=username
-DB_PASS=password
+DB_NAME=ushahidi
 DB_TYPE=MySQLi
-DB_USER=ushahidi
+DB_USER=username
```
Comment (rjmackay) [inline install/installing-on-linux.md:74]: @cmacdonell Do you think it makes more sense to just show `username` here? or use `ushahidi` for both? 
the GRANT command above shows creating both a DB and user named `ushahidi`.
My feeling is that we should either match the credentials we used above, or use a placeholder for db too `DB_NAME=database`

**Comment by rjmackay** [comment]
Nice catch :)

File: install/installing-on-linux.md (hunk 1 above)
Comment (cmacdonell) [inline install/installing-on-linux.md:74]: It makes sense either way really.  So I'll leave the decision to you.

File: install/installing-on-linux.md [hunk 2]
```diff
@@ -69,10 +69,10 @@ Now create a `.env` config file in the base of repository. Make sure that the da
...
+DB_PASS=password
 ```
```
Comment (cmacdonell) [inline install/installing-on-linux.md:76]: In that same vein, regarding the GRANT command, shouldn't 'set-a-custom-password-here' be the same as 'password' in DB_PASS or am I miss understanding?

GRANT ALL ON ushahidi.\* to ushahidi@localhost IDENTIFIED BY 'set-a-custom-password-here';

Comment (rjmackay) [inline install/installing-on-linux.md:76]: Correct. I'll sync the 2 up :)

**Pull request merged**